dxr/plugins/rust/__init__.py

```diff
@@ -1075,5 +1075,6 @@ def process_end_external_crates(args, tree):
 
 plugin = Plugin(filters=filters_from_namespace(filters.__dict__),
                 tree_to_index=TreeToIndex,
+                badge_colors={'rust':'burlywood'},
```

Comment:
I'd love to change this to 'rs' to shorten it and match the extension, but it would require a format bump. :-(